src/jit/lsra.cpp

```diff
@@ -6814,41 +6946,65 @@ LinearScan::insertCopyOrReload(GenTreePtr tree, RefPosition* refPosition)
         oper = GT_COPY;
     }
 
-    var_types treeType = tree->TypeGet();
+    // If the parent is a reload/copy node, then tree must be a multi-reg call node.
+    // Because multi-reg call node is the only node whose RefTypeDef positions get
```

Comment:
I would suggest saying "If the parent is already a reload/copy node, then tree must be a multi-reg call node that has already had one if its registers spilled." This will help clarify the scenario. Also, either begin the next sentence with "This is because" or don't start a new sentence.